addons/event_sms/data/sms_data.xml

```diff
@@ -5,15 +5,13 @@
         <field name="name">Event: Registration</field>
         <field name="model_id" ref="event.model_event_registration"/>
         <field name="body">{{ object.event_id.organizer_id.name or object.event_id.company_id.name or user.env.company.name }}: We are happy to confirm your registration for the {{ object.event_id.name }} event.</field>
-        <field name="lang">{{ object.partner_id.lang }}</field>
```

Comment:
Probably worth a migration to us the right value "'{{ object.event_id.lang or object.partner_id.lang }}" ?

Reply:
We can do that in a separate PR targeting the right version.